Backpack/src/main/java/net/skyscanner/backpack/rating/internal/RatingSelectors.kt

```diff
@@ -85,6 +89,10 @@ internal class RatingSelectors(
       colorMedium = it.getColor(R.styleable.BpkRating_ratingColorMedium, colorMedium)
       colorHigh = it.getColor(R.styleable.BpkRating_ratingColorHigh, colorHigh)
 
+      textColorLow = it.getColor(R.styleable.BpkRating_ratingTextColorLow, textColorLow)
```

Comment:
Let's use color state lists when possible. Also, can we just use colours directly instead of introducing new properties?